velox/functions/lib/tests/Re2FunctionsTest.cpp

```diff
@@ -461,40 +461,52 @@ TEST_F(Re2FunctionsTest, likePattern) {
 }
 
 TEST_F(Re2FunctionsTest, likeDeterminePatternKind) {
-  auto testPattern =
-      [&](StringView pattern, PatternKind patternKind, vector_size_t length) {
-        EXPECT_EQ(
-            determinePatternKind(pattern), std::make_pair(patternKind, length));
-      };
+  auto testPattern = [&](StringView pattern,
+                         PatternKind patternKind,
+                         vector_size_t length,
+                         StringView fixedPattern = "") {
+    PatternMetadata patternMetadata = determinePatternKind(pattern);
+    EXPECT_EQ(patternMetadata.patternKind, patternKind);
+    EXPECT_EQ(patternMetadata.length, length);
+    EXPECT_EQ(patternMetadata.fixedPattern, fixedPattern);
+  };
 
   testPattern("_", PatternKind::kExactlyN, 1);
```

Comment:
Add a test for %%% as Laith is suggesting.

Reply:
Done, thanks.